hl-roles/README.md

```diff
@@ -0,0 +1,106 @@
+- Curate job opportunities for students. 
+
+
+## Management Team
+
+### General Manager
+
+#### Responsibilities
+- Coordinate and facilitate initiatives from all Computer Science related entities, such as MCSS, DSC, and CS faculty
+- Create innovative incentives for new members to visit the HackLab 
+- Investigate the different sentiments surrounding the Computer Science Student Experience on campus and all digital platforms
+Correct the direction of HackLab when necessary
+- Interface with students and faculty through weekly sync ups via a report and an in-person discussion with MCS Department Chair regarding the status of the student landscape 
+- Maintain a public calendar on the HackLab website, Google Calendar, and Outlook Calendar for all CS related events across different entities 
+- Create opportunities to train and co-organize independent student initiatives, such as student-run workshops or looking for teammates for a project
+- Collaborate on Mental Health Support Initiatives with UTM Student Resources for professional aid such as UTM Wellness.
+- Interface with UTM Student Resources such as the Career Center
+Co-Host events to improve student experience such as a PoST and PEY Information Session with Academic & Professional Team
+- Support Operations Officer, Academic & Professional Team
+- Host general inquiry feedback office hours online and on campus
+#### Qualifications
+- A passion and a clear vision for improving the Computer Science Student community.
+- Compassion for understanding the landscape of the Computer Science Student experience from all its different communities. 
+- Excellent interpersonal and highly developed communication skills (verbal and written).
+- Detail-oriented: handle multiple projects at once and prioritize effectively.
+- Leadership acumen and the passion to apply it to empower students in their pursuit of a tech career.
+- A proven track record of one’s ability to scope workload.
+
+### Operation Officer
+
+#### Responsibilities
+- Define, enforce and maintain a HackLab etiquette. 
+- Coordinate and support General Manager, Academic & Professional Team 
+- Communicate state of HackLab, initiatives and team check-ins to MCS Department.
+- Maintain the HackLab website and regularly contribute new content.
+- Create promotional content for Hacklab.
+- Define requirements needed in order to be granted access to space. 
+- Coordinate with Andrew Wang to grant/revoke student access.
+- Maintain a room & equipment (VR/3D Printing) usage scheduling system.
+- Write business cases, manage hacklab finances, and interface with Donna Coulson.
+- Interface with UTM Resources and other departments. 
+- Actively seek out collaborations with other departments outside of MCS.
+
+#### Qualifications
+- A passion and a clear vision for improving the Computer Science Student community.
+- Compassion for understanding the landscape of the Computer Science Student experience from all its different communities. 
```

Comment:
Could be shortened to "An understanding of the computer science student experience from all of its communities.